src/runtime/runtime_tinygowasm_unknown.go

```diff
@@ -0,0 +1,50 @@
+//go:build wasm_unknown
+
+package runtime
+
+const (
+	stdout = 1
+)
+
+func putchar(c byte) {
+}
+
+func getchar() byte {
+	// dummy, TODO
+	return 0
+}
+
+func buffered() int {
+	// dummy, TODO
+	return 0
+}
+
+//go:linkname now time.now
+func now() (sec int64, nsec int32, mono int64) {
+	return 0, 0, 0
+}
+
+// Abort executes the wasm 'unreachable' instruction.
+func abort() {
+	trap()
+}
+
+//go:linkname syscall_Exit syscall.Exit
+func syscall_Exit(code int) {
+	//proc_exit(uint32(code))
+}
+
+// TinyGo does not yet support any form of parallelism on WebAssembly, so these
```

Comment:
Wasm doesn't provide any in the first place. There is a threading proposal but it only covers shared memory and atomic operations. Actually spawning the threads is left to the host environment, you can't do that from wasm.

Reply:
Comment changed. Thanks for the feedback!